src/ir/enum_ty.rs

```diff
@@ -49,6 +54,11 @@ impl Enum {
         &self.variants
     }
 
+    /// The special attributes of enumeration
+    pub fn attributes(&self) -> &Vec<Attribute> {
```

Comment:
That's fair, I think what we want is the offset for now. I'll see if I can see a more generic way of handling attributes that tokenizing them and evaluating them by hand, because making that work in every possible case is going to be a pain, over all if we take into account C++ `constexpr` functions and all that.

I think we definitely want to use llvm APIs for this.